src/DetailsView/tab-stops-minimal-requirement-header.scss

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../common/styles/colors.scss';
+@import '../common/styles/fonts.scss';
+
+.requirement-detail {
+    font-size: 14px;
+
+    padding: 16px 8px;
+
+    display: flex;
+    align-items: baseline;
+    text-align: left;
+
+    :global(.outcome-chip) {
+        vertical-align: middle;
+        margin-bottom: 2px;
+    }
+}
+
+.requirement-detail-description {
+    color: $secondary-text !important;
```

Comment:
Is the `!important` necessary? We try to not use them since those dependent on software that modifies color styles may run into issues.

Reply:
Same with below.